product_docs/docs/pgd/5/durability/administering.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: Administering
+navTitle: Administering
+---
+
+When running a PGD cluster with Group Commit, there are some things you need to
+be aware of when administering the system, such as how to safely shut down and 
+restart nodes.
+
+## Planned shutdown and restarts
+
+When using Group Commit with receive confirmations, take care
+with planned shutdown or restart. By default, the apply queue is processed
+prior to shutting down. However, in the `immediate` shutdown mode, the queue
+is discarded at shutdown, leading to the stopped node "forgetting"
+transactions in the queue. A concurrent failure of the origin node can
+lead to loss of data, as if both nodes failed.
```

Comment:
Should the last sentence be highlighted into a warning?

Reply:
I don't feel it should. This is discussing shutdowns and restarts and just citing the worst case.